### ISSUE TITLE
Following tutorial in docs, homepage.sass does not create CSS properties on correct Flutter widget

### ISSUE THREAD

**Issue by hcunninghampk** [flutter-view/flutter-view#38]
Hi @blueneogeo ,
I've been trying to follow the test drive tutorial in the flutter-view docs.  When I create the homepage.sass file (I've already created the homepage.pug which already generated the homepage.dart file) and flutter-view updates my homepage.dart file per the sass, flutter-view places the CSS properties on the Center() flutter widget, which is incorrect.  The CSS properties from homepage.sass for the .greeting class are text CSS properties, which means in flutter they have to go in a TextStyle() child widget of a Text() widget.  And now, the app won't even run.  

These CSS properties created by flutter-view in homepage.dart are not in correct Flutter syntax either.  The following is the Flutter widget created/edited by flutter-view from the homepage.sass file:

`
Center( // project://lib\screens\homepage\homepage.pug#5,9
      fontSize: '24px',
      fontWeight: 'bold',
      child: Text( 
        'Hello Flutter-View World!',
      ),
      color: Colors.blue,
    ),
`     

But, this snippet below is what the correct Flutter syntax should be:

`
Center( // project://lib\screens\homepage\homepage.pug#5,9
      child: Text( 
        'Hello Flutter-View World!',
        style: TextStyle(
          fontSize: 24,
          fontWeight: FontWeight.bold,
          color: Colors.blue
        ),
      ),
    ),
`

The CSS generated and added to the homepage.dart by the homepage.sass and flutter-view is not correct and doesn't work.

**Comment by hcunninghampk** [comment]
Here is a link to my GitHub repo for the flutter-view test drive: [https://github.com/hcunninghampk/flutterview_testdrive](https://github.com/hcunninghampk/flutterview_testdrive).  I didn't bother creating a feature branch yet, everything is in the main  branch so far, since this repo is only for this tutorial.

**Comment by hcunninghampk** [comment]
BTW, @blueneogeo , I love this idea; I just wish it worked correctly.  But, it's an amazing idea and I would love to use it.

**Comment by hcunninghampk** [comment]
Oh!  My bad...  I'm not familiar with pug and a newbie at sass.  I didn't realize in the pug file the .greeting has to go on a separate line.  I appended it to "center(as='body')".  I moved the .greeting to its own indented line in the pug and now the generated dart edits are almost perfect!  I just had to take the "px" out of   fontSize: 24px.  But, the CSS props are now correctly on a TextStyle child widget, even the fontWeight prop is correctly set to FontWeight.bold.  This is pretty awesome.  

I'll close this issue.
